oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -26,15 +26,38 @@ import "rc-slider/assets/index.css";
 
 const MODEL_CLASSES_WITH_QUIVER = Object.freeze(["Mercator"]);
 
-function DatasetSelector(props) {
+function DatasetSelector({
+  onUpdate,
+  id,
+  variables,
+  multipleVariables = false,
+  showQuiverSelector = true,
+  showTimeRange = false,
+  showDepthSelector = true,
+  showVariableRange = true,
+  showAxisRange = false,
+  showVariableSelector = true,
+  showDepthsAll = false,
+  horizontalLayout = false,
+  options,
+  mountedDataset,
+  mapSettings,
+  showTimeSlider,
+  compareDatasets,
+  showCompare,
+  action,
+  t,
+}) {
+  // Using t as the translation function instead of __
+  const __ = t;
```

Comment:
If we're including t in the component definition then it'll be more readable just to use `t` instead of `__` throughout. Just be sure to remove the plugin definition from the webpack config.